tests/framework-tests/preferences/js/SeparatedPanelPrefsEditorTests.js

```diff
@@ -81,6 +81,12 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         var prefsEditor = separatedPanel.prefsEditor;
         jqUnit.assertEquals("Reset button is invisible", false, $(".flc-prefsEditor-reset").is(":visible"));
         fluid.tests.prefs.assertPresent(prefsEditor, fluid.tests.prefs.expectedComponents["fluid.prefs.separatedPanel"]);
+
+        var toggleButtonAriaPressedState = separatedPanel.slidingPanel.locate("toggleButton").attr("aria-pressed");
+        var ariaExpandedState = separatedPanel.locate("iframe").attr("aria-expanded");
+
+        jqUnit.assertEquals("Show/hide button has correct aria-pressed", "false", toggleButtonAriaPressedState);
+        jqUnit.assertEquals("Panel has correct aria-expanded", "false", ariaExpandedState);
```

Comment:
These test the string associated with the aria attribute either "true" or "false", not whether or not the attribute value exists.